index.js

```diff
@@ -41,7 +50,19 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+function getHighestMetascore(movies) {
+  if (!movies.length) {
+    return 0;
+  }
+  let metascore = ~~movies[0].metascore;
```

Comment:
I love `~~` but this is a rare sight to see in JS. ` Math.floor()` might be more easily read solution.